pyminion/game.py

```diff
@@ -232,6 +243,14 @@ def is_over(self) -> bool:
         return False
 
     def play_turn(self, player: Player) -> None:
+        # if player played Possession while being possessed, take the extra turn
+        # before their main turn
```

Comment:
I had to google this, what a rabbit hole: https://boardgames.stackexchange.com/questions/10/if-possession-is-played-while-executing-another-possession-who-takes-the-next-t